stable/kubewatch/Chart.yaml

```diff
@@ -1,15 +1,18 @@
 name: kubewatch
-version: 0.3.1
+version: 0.3.2
```

Comment:
seems like quite a lot is changing shouldnt this be at least 0.4.0

Reply:
probably yes as I am adding new functionality